content/docker-hub/access-tokens.md

```diff
@@ -5,8 +5,8 @@ description: Learn how to create and manage your personal Docker Hub access toke
 keywords: docker hub, hub, security, PAT, personal access token
 ---
 
-If you are using the [Docker Hub CLI](https://github.com/docker/hub-tool#readme)
-tool (currently experimental) to access Hub images from the Docker CLI, you can create personal access tokens (PAT) as alternatives to your password.
+If you're using the [Docker Hub CLI](https://github.com/docker/hub-tool#readme)
```

Comment:
Note to reviewer: do we need to link to this or can we just refer to Docker CLI?

Reply:
`hub-tool` is a different tool, not included in the main CLI.

TBH I'm not convinced that we should even bring up `hub-tool` in our docs. It doesn't look like it's maintained. @RomainBelorgey @silvin-lubecki @chris-crone Do you know the status of `hub-tool`?

Reply:
Unfortunately yes this tool is not maintained anymore, but it should. I'd really like to transform it as a `docker hub` cli plugin.

Reply:
@silvin-lubecki I'd like that too!

Reply:
Created a new PR for this due to all the security IA updates! New PR is here, I omitted the reference to this tool and just simplified the intro: https://github.com/docker/docs/pull/18458

Reply:
Thanks @stephaurelio